integrations/docker/images/chip-build/Dockerfile

```diff
@@ -120,6 +121,32 @@ RUN set -x \
     && rm -rf bloaty \
     && : # last line
 
+# Need newer version of include-what-you-use
+RUN set -x \
+    && apt-get update \
+    # Install build and runtime requirements for IWYU
+    && DEBIAN_FRONTEND=noninteractive apt-get install -fy --fix-missing clang-12 libclang-12-dev \
+    # Build and install IWYU
+    && git clone --depth=1 --branch=clang_12 https://github.com/include-what-you-use/include-what-you-use.git \
+    && mkdir -p include-what-you-use/build \
+    && cd include-what-you-use/build \
+    && cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_PREFIX_PATH=/usr/lib/llvm-12 -DIWYU_LINK_CLANG_DYLIB=OFF .. \
+    && make -j8 \
+    && make install \
+    # Save Clang includes, so we can restore them after build dependencies cleanup
```

Comment:
Investigate how to do it in a more effective way.
NOTE: Do not merge until this comment will be resolved.